docs/source/contents/4_draft_mapping.rst

```diff
@@ -2152,6 +2152,622 @@ Drop all values in mods:part.
+| dcterms:tableOfContents       | String Literal     | N/A   | A String Literal that represents constituent parts of a resource.                          |
++-------------------------------+--------------------+-------+--------------------------------------------------------------------------------------------+
+| dcterms:hasVersion            | URI                | N/A   | A String Literal that represents constituent parts of a resource.                          |
++-------------------------------+--------------------+-------+--------------------------------------------------------------------------------------------+
+| dbo:collection                | String Literal     | N/A   | A String Literal that represents the physical archival collection the resource belongs to. |
++-------------------------------+--------------------+-------+--------------------------------------------------------------------------------------------+
+| dbo:isPartOf                  | URI                | N/A   | A URI that represents the physical archival collection a resource belongs to.              |
++-------------------------------+--------------------+-------+--------------------------------------------------------------------------------------------+
+| opaque:memberOfArchivalSeries | URI/String Literal | N/A   | A URI or String Literal that indicates the archival series a resource belongs to.          |
++-------------------------------+--------------------+-------+--------------------------------------------------------------------------------------------+
+| opaque:sheetMusic_hostItem    | URI/String Literal | N/A   | A URI or String Literal that indicates the host item for the resource being described.     |
++-------------------------------+--------------------+-------+--------------------------------------------------------------------------------------------+
+| relators:[term]               | URI/String Literal | N/A   | Use with a role from MARC Code List of Relators role terms.                                |
+|                               |                    |       | Value is either text or URI from a controlled vocabulary (like                             |
+|                               |                    |       | Library of Congress Name Authority File).                                                  |
++-------------------------------+--------------------+-------+--------------------------------------------------------------------------------------------+
+
+relatedItem as simple structural metadata
+-----------------------------------------
+Use Case
+^^^^^^^^
+`relatedItem`, with and without attributes, is used in a number of collections to express structural relationships. These relationships will be handled/expressed by default behavior in our next-gen DAMS. In the case of `relatedItem/abstract`, the values present should be handled at the collection level.
```

Comment:
If you want, you could also include here a little bit about how these values are currently displayed or shared in the Use Case section. Currently we do have Solr fields for these values (utk_mods_relatedItem_titleInfo_title_ms) and we show the collection title in the Click for Details section using this field. Going forward we can duplicate this just using the relationship in the DAMS or just remove it as there are usually many other indicators of the larger collection on an item page.